Update the page on `-push-to-cloud` cmd (#3047)
neo4j/docs-operations: modules/ROOT/pages/tools/neo4j-admin/push-to-cloud.adoc

Author: I think we need to remove this label, as this is not available in the current Aura

Author: I replaced this label with 'Not available on Aura'

Author: ```suggestion
:page-role: not-on-aura
```

:description: How to import a database from an existing Neo4j instance into Neo4j Aura using `neo4j-admin push-to-cloud`.
:page-role: not-on-aura
[[neo4j-admin-push-to-cloud]]
= Upload to Neo4j AuraDB

[WARNING]
====
It is impossible to transfer your self-managed 4.4 Neo4j directly to Neo4j Aura.

> First, you must upgrade your Neo4j to 5.26 LTS minimum or to the link:{neo4j-docs-base-uri}/operations-manual/current/[latest calendar version of Neo4j server]. +

Author: ```suggestion
You must upgrade your Neo4j to 5.26 LTS minimum.
After that, you can upgrade Neo4j to any release of 2025-2026 cycle.
See the link:{neo4j-docs-base-uri}/operations-manual/current/[latest calendar version of Neo4j server]. +
```

[[neo4j-admin-push-to-cloud]]
= Upload to Neo4j AuraDB

[WARNING]
====
It is impossible to transfer your self-managed 4.4 Neo4j directly to Neo4j Aura.

> First, you must upgrade your Neo4j to 5.26 LTS minimum or to the link:{neo4j-docs-base-uri}/operations-manual/current/[latest calendar version of Neo4j server]. +

Author: Should we recommend upgrading to the latest Neo4j version or just say that you can upgrade to any 2025.x or 2026.x release?

For details on upgrade and migration options, see link:{neo4j-docs-base-uri}/upgrade-migration-guide/current/[Upgrade and migration guide].
====

The following table shows the compatibility between the dump version that you want to upload and the version of the Neo4j Aura instance.

[options="header" cols="50%, 50%"]
|===
| Dump version
| Aura version

| The minimum version of self-managed Neo4j must be 5 LTS (5.26.x)
| Corresponds to the latest version of Neo4j server. +

> See link:{neo4j-docs-base-uri}/operations-manual/current/[] for details.

Author: ```suggestion
See link:{neo4j-docs-base-uri}/operations-manual/current/[Operations manual for the current calendar version] for details.
```
